homeassistant/components/tibber/sensor.py

```diff
@@ -42,52 +43,89 @@
 SIGNAL_UPDATE_ENTITY = "tibber_rt_update_{}"
 
 RT_SENSOR_MAP = {
-    "averagePower": ["average power", DEVICE_CLASS_POWER, POWER_WATT],
-    "power": ["power", DEVICE_CLASS_POWER, POWER_WATT],
-    "minPower": ["min power", DEVICE_CLASS_POWER, POWER_WATT],
-    "maxPower": ["max power", DEVICE_CLASS_POWER, POWER_WATT],
+    "averagePower": ["average power", DEVICE_CLASS_POWER, POWER_WATT, None],
+    "power": ["power", DEVICE_CLASS_POWER, POWER_WATT, None],
+    "minPower": ["min power", DEVICE_CLASS_POWER, POWER_WATT, None],
+    "maxPower": ["max power", DEVICE_CLASS_POWER, POWER_WATT, None],
     "accumulatedConsumption": [
         "accumulated consumption",
         DEVICE_CLASS_ENERGY,
         ENERGY_KILO_WATT_HOUR,
+        STATE_CLASS_MEASUREMENT,
     ],
     "accumulatedConsumptionLastHour": [
         "accumulated consumption last hour",
         DEVICE_CLASS_ENERGY,
         ENERGY_KILO_WATT_HOUR,
+        STATE_CLASS_MEASUREMENT,
```

Comment:
Last hour should not be a measurement as it doesn't contain the present value.

See also the extra clarification for the docs pending in this PR https://github.com/home-assistant/developers.home-assistant/pull/950/files

Reply:
Actually, the name was incorrect, renamed it to current hour now.
It will not be a breaking change since the name was introduced in https://github.com/home-assistant/core/pull/50418

Reply:
So, this is still an hourly aggregation of data, right?
Thus, should not have this state classification.

Reply:
If I understood @emontnemery correctly hourly aggregated data should have STATE_CLASS_MEASUREMENT 

> Note that for accumulated values you should also set the last_reset property.
https://github.com/home-assistant/core/pull/50951#discussion_r637928997

Reply:
Yes, if this is the energy consumption, reset hourly?

Reply:
Yes, so at 13:25, it will show energy consumed between 13:00 and 13:25